recipes/asciidoctor-reducer/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "asciidoctor-reducer" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 9c6a132414846aaeb4620dceb9687ed3f0e827135b3514c2fc46d8564f9a79b9
+  - path: asciidoctor-reducer.bat
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies -n $PREFIX/bin {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - cp -v asciidoctor-reducer.bat $PREFIX/bin/asciidoctor-reducer.bat
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - {{ pin_compatible('ruby', max_pin='x.x') }}
+    - asciidoctor >=2.0
+
+test:
+  commands:
+    - asciidoctor-reducer -h
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
```

Comment:
Can you remove the excess of jinja? It makes the URLs unusable and serves no purpose, we recommend it just for the version and the name in rare places, like PyPI URLs. Also not that the future recipe format will remove jinja, so you are already future proofing it.